src/v3/src/types/schema.ts

```diff
@@ -432,7 +424,7 @@ export interface AccordionLayout {
   elements: AccordionPanelElement[];
 }
 
-export interface StepperButtonElement {
+export interface StepperButtonElement extends UISchemaElement {
```

Comment:
These need to extend UISchemaElement because there are properties (i.e. focus) needed in dependent code.